Cargo.toml

```diff
@@ -28,7 +28,7 @@ tiny-bip39 = { version = "^0.7", optional = true }
 
 # Platform-specific dependencies
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
-tokio = { version = "0.2", features = ["rt-core"] }
+tokio = { version = "0.3", features = ["rt"] }
```

Comment:
[Here are tokio's old features](https://docs.rs/tokio/0.2.23/tokio/index.html#feature-flags), and [here are tokio's new features](https://docs.rs/tokio/0.3.3/tokio/index.html#feature-flags).

It seems like `rt-core` became `rt`.